src/GovUk.Education.ExploreEducationStatistics.Data.Api/Controllers/DownloadController.cs

```diff
@@ -40,14 +41,27 @@ public async Task<ActionResult> GetChartFile(string publication, string release,
         private async Task<ActionResult> GetFile(string publication, string release, ReleaseFileTypes type,
             string filename)
         {
-            if (!_fileStorageService.FileExistsAndIsReleased(PublicFilesContainerName,
-                PublicReleasePath(publication, release, type, filename)))
+            var isReleased = await _fileStorageService.IsBlobReleased(
+                containerName: PublicFilesContainerName,
```

Comment:
I've noticed that you are adding these redundant argument name identifiers in lots of places. Why is that? I think you can get the IDE to overlay these if you find them helpful.

Reply:
It's mostly because we're usually working with lots of string parameters in these methods and I want us to be careful not to get these wrong if the argument order changes during a refactor. The nice thing about named parameters is that these errors are easily identified at compile time. 

I would encourage you to consider using them as well as much as possible.